docs/changehistory/NextVersion.md

```diff
@@ -3,10 +3,50 @@
 Table of contents:
 
 - [@itwin/appui-react](#itwinappui-react)
+  - [Deprecations](#deprecations)
   - [Changes](#changes)
 
 ## @itwin/appui-react
 
+### Deprecations
+
+- Deprecated `SignInProps` instead use `React.ComponentProps<typeof SignIn>`. [#991](https://github.com/iTwin/appui/pull/991)
+- Deprecated `AccuDrawDialogProps` instead use `React.ComponentProps<typeof AccuDrawDialog>`. [#991](https://github.com/iTwin/appui/pull/991)
+- Deprecated `AccuDrawFieldContainerProps` instead use `React.ComponentProps<typeof AccuDrawFieldContainer>`. [#991](https://github.com/iTwin/appui/pull/991)
+- Deprecated `AccuDrawInputFieldProps` instead use `React.ComponentProps<typeof AccuDrawInputField>`. [#991](https://github.com/iTwin/appui/pull/991)
```

Comment:
Not all of these types are from the `@itwin/appui-react` package.

Reply:
Split props by the packages.